js/ide.js

```diff
@@ -2127,6 +2127,29 @@ var ide = new function() {
         encodeURIComponent(query) +
         "&target=compact";
 
+      function constructOverpassQLUrl(query) {
+        // remove /* */ comments from query
+        query = query.replace(/\/\*(.|\n)*?\*\//g, "");
+        // replace bbox with south west north east
+        query = query.replace(/{{bbox}}/g, "{south},{west},{north},{east}");
+        // replace //  comments from query
+        query = query.replace(/\/\/.*/g, "");
+        // removes indentation
+        query = query.replace(/\n(\s|\t)+/g, "");
```

Comment:
doesn't `\s` also match tabulators? so simply `replace(/\n\s+/g, "")` would be enough, right? Also, this replace could be combined with the next line by writing `replace(/\n\s*/g, "")`, right?